release/ray_release/command_runner/command_runner.py

```diff
@@ -3,16 +3,30 @@
 
 from ray_release.cluster_manager.cluster_manager import ClusterManager
 from ray_release.file_manager.file_manager import FileManager
+from ray_release.reporter.artifacts import DEFAULT_ARTIFACTS_DIR
 from ray_release.util import exponential_backoff_retry
 from click.exceptions import ClickException
 
 
 class CommandRunner(abc.ABC):
+    """This is run on Buildkite runners."""
+
+    # the directory for runners to dump files to (on buildkite runner instances).
+    # Write to this directory. run_release_tests.sh will ensure that the content
+    # shows up under buildkite job's "Artifacts" UI tab.
+    _DEFAULT_ARTIFACTS_DIR = DEFAULT_ARTIFACTS_DIR
+
+    # the artifact file name put under s3 bucket root.
+    # AnyscalejobWrapper will upload user generated artifact to this path
+    # and AnyscaleJobRunner will then download from there.
+    _USER_GENERATED_ARTIFACT = "user_generated_artifact"
```

Comment:
can we also move `result_output_json` and `metrics_output_json` (with consistent capitalization)?

Reply:
done. @Yard1 just wanted result_output_json and metrics_output_json  to be moved to class var.